ReadExcelFile.py

```diff
@@ -19,6 +19,15 @@
 #Print the cell value
 print(my_val)
 
+#Read in values of cells C1 through C5 as list (array).
```

Comment:
This doesn't actually read the values, but creates an array of cell objects. That's the reason why below, in line 29, you have to say `.value`. Try to make a list of values, i.e. integers.

The reason I'm emphasizing this is that when you use cell objects you don't know what it might be doing behind the scenes. The hidden complexity is important when you're reading an Excel sheet (otherwise you'd have to parse the file yourself and do all kinds of annoying BS you don't want to care about), but it's best to handle simple objects (such as integers) as much as possible.

Also, once you have a list of values you can just call print(MyList) on it and not have to use a for loop.